### DISCUSSION TITLE
FortiSwitch FS-124F series?

### DISCUSSION THREAD

**Discussion by nickolas1969** [netbox-community/devicetype-library#1666] [General]
I love the completeness of this distro device types, we just started buying Fortinet gear and the one switch type we got isn't in this list, curious if anyone has it? FS-124F-FPOE ?

**Comment by danner26** [comment]
This is widely a community contributed repo. If you'd like, you can create it yourself and submit a PR. I would be happy to review it and suggest any changes before it it merged

**Reply by nickolas1969** [reply]
Ok, still trying to figure out how to do git submissions.... I did take the 224D and edit all the specs to match what the 124F-FPOE has, so power draw, 4 10G uplinks instead of 1G. 

`---
manufacturer: Fortinet
model: FortiSwitch 124F-FPOE
slug: fortinet-fs-124f-fpoe
part_number: FS-124F-FPOE
u_height: 1
weight: 8.42
weight_unit: lb
is_full_depth: false
airflow: side-to-rear
console-ports:
  - name: Console
    type: rj-45
module-bays:
  - name: PS1
    type: iec-60320-c14
    maximum_draw: 451.3
    allocated_draw: 449.8
interfaces:
  - name: port1
    type: 1000base-t
  - name: port2
    type: 1000base-t
  - name: port3
    type: 1000base-t
  - name: port4
    type: 1000base-t
  - name: port5
    type: 1000base-t
  - name: port6
    type: 1000base-t
  - name: port7
    type: 1000base-t
  - name: port8
    type: 1000base-t
  - name: port9
    type: 1000base-t
  - name: port10
    type: 1000base-t
  - name: port11
    type: 1000base-t
  - name: port12
    type: 1000base-t
  - name: port13
    type: 1000base-t
  - name: port14
    type: 1000base-t
  - name: port15
    type: 1000base-t
  - name: port16
    type: 1000base-t
  - name: port17
    type: 1000base-t
  - name: port18
    type: 1000base-t
  - name: port19
    type: 1000base-t
  - name: port20
    type: 1000base-t
  - name: port21
    type: 1000base-t
  - name: port22
    type: 1000base-t
  - name: port23
    type: 1000base-t
  - name: port24
    type: 1000base-t
  - name: port25
    type: 10gbase-x-sfpp
  - name: port26
    type: 10gbase-x-sfpp
  - name: port27
    type: 10gbase-x-sfpp
  - name: port28
    type: 10gbase-x-sfpp`

**Reply by danner26** [reply]
To create a PR, first fork this repo to your GH. Then create a new branch, and add the definition file. After committing that file, you can create a PR from your new branch to the community repo and we will review it for merging
